drivers/watchdog/Kconfig.andes_atcwdt200

```diff
@@ -0,0 +1,15 @@
+# Kconfig Andes Watchdog configuration options
+#
+# Copyright (c) 2023 Andes Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config WDT_ANDES_ATCWDT200
+	bool "Andes Watchdog driver"
+	default y
+	depends on DT_HAS_ANDESTECH_ATCWDT200_ENABLED
+	select COUNTER
+	help
+	  Enable driver for the Andes Watchdog driver.
+	  Says n if not sure.
```

Comment:
Isn't it the custom to state "Say `<default>` if not sure"? It's kinda confusing to have default `y` and suggest `n` if unsure.

Reply:
OK, I got it.
I think this sentence is redundant, because there is default value, so I have removed it.
Thank you.